src/customdocument.cpp

```diff
@@ -154,13 +165,20 @@ bool CustomDocument::openLinkAtCursorPosition()
         if (isFileUrl) {
             QString trimmed = urlString.mid(7);
             if (!QFile::exists(trimmed)) {
-                qDebug() << __func__ << "file does not exist:" << urlString;
+                qDebug() << __func__ << ": File does not exist:" << urlString;
+                // show a message box
+                QMessageBox::warning(
```

Comment:
I'm showing a messagebox here, because before it might not have been clear to the user that the file doesn't exist.

Reply:
Maybe we should provide a custom style for messageboxes (in a separate PR)?

Reply:
I think we can stick with native one hah (less headache, and it should look good, at least on macOS it looks good).
But I I think we don't need to provide the name of the file. Just "This file doesn't exist." should be good enough, the link is already shown to the user in front of him.

Reply:
I think it's useful to have the filename in the messagebox, because if something goes wrong with the link grabbing (for instance, maybe a bug could occur where only a part of the link is extracted), then the user would easily see why it went wrong. Or maybe the user might think they clicked the wrong link by accident and now gets confirmation for which link they clicked.

Or we could just show the clicked link in the terminal logs (like we do now) and I can change the messagebox text.
Your call.

Reply:
Ah, well that makes sense. I tried to open a very long link that didn't work (on purpose) and it showed a very big message box, this is why I said it. But then, I agree with your approach. 👍